charts/argocd-apps/Chart.yaml

```diff
@@ -19,3 +19,6 @@ annotations:
   artifacthub.io/changes: |
     - kind: fixed
       description: not rendering empty app description
+  artifacthub.io/changes: |
+    - kind: added
+      description: allow setting applicationset finalizers
```

Comment:
line 19..21 needs to be removed

Reply:
corrected.

Reply:
Restored wrong deletion. Now the Chart changelog reflects current release changes.